lib/open_weather/endpoints/stations.rb

```diff
@@ -30,6 +30,11 @@ def delete_station(id)
         nil
       end
 
+      def create_measurements(measurements, options = {})
+        post('measurements', options.merge(body: measurements))
+        nil
```

Comment:
So that API returns a 204 instead of what it has created, annoying :(

Reply:
And it is a "bulk" endpoint as well :/